### PR TITLE
Issue 21

### PR DESCRIPTION
**Pull request by DrCherry**
This is a pretty major change to output.  I've replaced almost all of the print() functions with logging.  There's now a log file created called instabot.log which ends up in the examples directory.  

Of course the log levels can be adjusted per the module but I have everything going to info and warn levels.

### CODE REVIEW

File: instabot/api/prepare.py [hunk 1]
```diff
@@ -23,23 +24,25 @@ def check_secret():
                     login = f.readline().strip()
                     password = f.readline().strip()
                     if len(login) < 4 or len(password) < 6:
-                        print("Data in 'secret.txt' file is invalid. "
+                        
+                        logging.warning("Data in 'secret.txt' file is invalid. "
```
Comment (DrCherry) [inline instabot/api/prepare.py]: This needs to be tested.  I'm not sure if logging will even work here.

Comment (DrCherry) [reply instabot/api/prepare.py]: Yeah.  Good news is that' the prompts are going to the log file.  Bad news is that they're going to the log file! :dagger:

**Comment by DrCherry** [comment]
log file looks like

```
2017-02-13 20:39:26,033 Instabot Started
2017-02-13 20:39:37,744 Login success!
2017-02-13 20:39:37,744 Going to like media with hashtag #avantgarde
2017-02-13 20:39:38,956     Going to like 15 medias.

```

**Comment by DrCherry** [comment]
Console output:

```
2017-02-13 20:39:37,744 - [instabot] - INFO - Going to like media with hashtag #avantgarde
2017-02-13 20:39:38,956 - [instabot] - INFO -     Going to like 15 medias.
100%|█████████████████████████████████████████████████████████████████████| 15/15 [01:41<00:00,  8.38s/it]
2017-02-13 20:41:20,025 - [instabot] - INFO -     DONE: Total liked 15 medias. 
2017-02-13 20:41:20,026 - [instabot] - INFO - Going to comment medias by avantgarde hashtag

```

**Comment by ohld** [comment]
AWESOME!

**Pull request merged**